projects/bp-gallery/cypress/e2e/picture_uploading_and_tagging.cy.ts

```diff
@@ -0,0 +1,112 @@
+import { login, logout } from '../utils/login-utils';
+
+describe('picture uploading and tagging', () => {
+  before(() => {
+    cy.visit('http://localhost:3000/browse');
+    login();
+    cy.get('.nav-bar').contains('Mehr...').click();
+    cy.get('.MuiPaper-root').contains('Collections').click();
+    cy.contains('add').click();
+    cy.contains('Neue Collection erstellen').click();
+    cy.get('.MuiDialogContent-root').find('input').type('TestCollection');
+    cy.get('.MuiButton-root').contains('Bestätigen').click();
+    cy.get('.nav-bar').contains('Mehr...').click({ force: true });
+    cy.get('.MuiPaper-root').contains('Uploads').click();
+  });
+  after(() => {
+    cy.wait(1000);

```

Comment:
wieso wartet ihr am anfang bei after?

Reply:
Probably because of loading times. But I'm not sure why here exactly.